README.md

```diff
@@ -53,5 +53,19 @@ See [contrib/tools/workflowcheck](contrib/tools/workflowcheck) for a tool to det
 ## Contributing
 We'd love your help in making the Temporal Go SDK great. Please review our [contribution guidelines](CONTRIBUTING.md).
 
+## Go build and run tags
+
+Prior to SDK version v1.26.0 our protobuf code generator allowed invalid UTF-8 data to be stored as proto strings. This isn't actually allowed by the proto3 spec, so if you're building our SDK manually and think you may store arbitrary binary data in our strings you should set `-tags protolegacy` when building against our SDK.
+
+Example:
+
+``` shell
+$ go build -tags protolegacy myworker/main.go
+```
+
+If you see don't but see an error like `grpc: error unmarshalling request: string field contains invalid UTF-8` then you will need to enable this when building your code.
```

Comment:
"If you see don't but see an error like" -> "If you see an error like"

Reply:
🤦 I'll fix that, thanks Quinn.